Changelog.md

```diff
@@ -5,6 +5,20 @@ All notable Changes to the Julia package `Manopt.jl` will be documented in this
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.4.63] unreleased
+
+### Added
+
+* `:reinitialize_direction_update` option for quasi-Newton behavior when the direction is not a descent one. It is now the new default for `QuasiNewtonState`.
+
```

Comment:
We could add a short entry here that the quasi Newton update rules are now initialised upon start of a solver with the new internal function `initialize_update!`; just to keep that complete in case someone want to add a new rule.

Reply:
Done.